templates/vue/src/components/tyde/TydeActivitySummary.vue

```diff
@@ -49,9 +49,6 @@ export default {
       }))
     },
   },
-  mounted() {
-    const entries = Object.entries(this.activity.entries)
-  },
```

Comment:
This is different from the master version because it was causing a lint error that existed in TYDE only.